src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -118,10 +118,16 @@ class AsyncJobs(threading.Thread):
     def __init__(self, volume_client, name_pfx, nr_concurrent_jobs):
         threading.Thread.__init__(self, name="{0}.tick".format(name_pfx))
         self.vc = volume_client
-        # queue of volumes for starting async jobs
+        # deque where each member is a string and where each string is name of
```

Comment:
Just `deque of volume names` ?

Reply:
Updated. I've kept it a bit verbose so future readers don't have to guess.